engine/Cargo.toml

```diff
@@ -19,6 +19,7 @@ crate-type = ["cdylib", "rlib"]
 aurora-engine-types = { path = "../engine-types", default-features = false }
 aurora-engine-sdk = { path = "../engine-sdk", default-features = false }
 aurora-engine-precompiles = { path = "../engine-precompiles", default-features = false }
+aurora-engine-transactions = { path = "../engine-transactions", default-features = false }
```

Comment:
Should also include this new crate in the `std` features list below (outside the diff, so I can't comment on it directly)